polaris-cli/bin/dev

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
```

Comment:
If you follow the suggestions that I gave you, I don't think you need any executable in this repository besides the one that comes with the `@shopify/cli` package.